### DISCUSSION TITLE
minIO S3 previously uploaded files are not displaying in Assets

### DISCUSSION THREAD

**Discussion by IVoyt** [statamic/cms#10293] [Q&A] [answered]
I already was creating a support ticket via website but haven't received a confirmation message. So if it is a duplicate - I'm sorry.

The problem - I have a local S3 working under minIO and there are previously uploaded files in it. I've connected it in Statamic, I can see folders structure, but I can't see these previously uploaded files in end folder. While if I upload any file via Statamic Assets I can see the uploaded file

<img width="1969" alt="image" src="https://github.com/statamic/cms/assets/10925772/cb431424-1e1c-41e6-93f7-f7d563f0b5e6">
<img width="2312" alt="image" src="https://github.com/statamic/cms/assets/10925772/a51eca76-bde4-4edd-9b4c-58b6b73266c3">

I have checked if Storage provider can read the folder with the following code
```php
dump(\Illuminate\Support\Facades\Storage::files('images/general/2024/05/15'));
```
and it returns all the files stored in the path. So it is not problem with flysystem s3 driver

Also there is some strange behaviour - after uploading file via Statamic Assets in the S3 end folder appeared a new folder with same name as end folder, and if remove the created folder it removes end folder as well. I understand it is not related to Statamic but it looks weird

**Comment by edalzell** [comment]
Clear your cache, it should pop up then

**Reply by IVoyt** [reply]
> Clear your cache, it should pop up then

Unfortunately neither `stache:clear` nor `cache:clear` helped

**Reply by IVoyt** [reply]
I think I found the problem - Statamic is searching for records in `assets_meta` table. In my case I have only one record - an uploaded file via Statamic Assets and therefore I can't see anything but this uploaded file.

So there is the question - how can I fill the table if I already have previously uploaded files in my local S3?

**Reply by ryanmitchell** [reply, marked as answer]
`php please eloquent:sync-assets`

**Reply by IVoyt** [reply]
thanks for the help
